clang/test/CodeGenCXX/cxx23-fp-ext-std-names-p1467r9.cpp

```diff
@@ -0,0 +1,563 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --function-signature
+// RUN: %clang_cc1 -std=c++23 -triple x86_64-unknown-unknown -target-feature +fullbf16 -emit-llvm %s -o - | FileCheck %s  --check-prefix=CHECK
```

Comment:
x86 is unfortunately pretty tame as far as testing these rules goes. You should also test on:
* a target where double = long double = IEEE double-precision
* a target where long double = IEEE quad-precision
* a target where long double = `ppc_fp128`